src/template.js

```diff
@@ -8,6 +8,8 @@ const makeInteger = require('makeInteger');
 
 const TEMPLATE_VERSION = '1.1.0';
```

Comment:
We should bump this to `2.0.0` imo since it's a big change. (It's used to track Algolia user agents.)

Reply:
Yeah I was going to bump it as a separate release PR. But do you think it's major bump? Big change but nothing breaks.

Reply:
The default cookie behavior changes because we now rely on Search Insights 2, right?